content/applications/services.rst

```diff
@@ -11,3 +11,4 @@ Services
     services/timesheets
     services/field_service
     services/helpdesk
+    services/planning
```

Comment:
I think it'd make more sense to have Planning either after timesheets, or after field services, but not at the end, after helpdesk.

Reply:
Done